infrastructure/src/main/java/org/corfudb/infrastructure/BatchProcessor.java

```diff
@@ -129,6 +131,10 @@ private void processor() {
                     log.warn("Shutting down the write processor");
                     streamLog.sync(true);
                     break;
+                } else if (streamLog.quotaExceeded() && currOp.getMsg().getPriorityLevel() != PriorityLevel.HIGH) {
+                    // TODO(Maithem): should the exception include the used/limit metrics?
+                    currOp.getFutureResult().completeExceptionally(new QuotaExceededException());
+                    log.warn("batchprocessor: dropping msg {}", currOp.getMsg());
```

Comment:
Add reason for "dropping msg" this log, also s/batchprocessor/batchProcessor

Reply:
Done.